cves/kernel/CVE-2016-2184.yml

```diff
@@ -189,7 +205,9 @@ specification:
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
   note:
-  answer:
+    This is because the application does not check for endpoints which is
+    a very realistic possiblity that can occur.
```

Comment:
This is asking about a specific specification being violated. Are you referring to a particular specification here?